packages/xstate-wallet/src/store/store.ts

```diff
@@ -324,6 +327,27 @@ export class XstateStore implements Store {
 
   private nonceKeyFromAddresses = (addresses: string[]): string => addresses.join('::');
 
+  async getForceMoveTransactionData(channelId: string): Promise<TransactionRequest> {
```

Comment:
This is the minimal API to add to `Store` without changing any code inside `nitro-protocol`.